packages/util/package.json

```diff
@@ -17,7 +17,7 @@
     "@babel/preset-typescript": "7.6.0",
     "@types/jest": "24.0.15",
     "jest": "24.9.0",
-    "typescript": "3.6.4"
+    "typescript": "^3.9.7"
```

Comment:
hmm what would you think of pinning it to 3.9.10 for stability? leave it up to you, also ok if you think it's fine as-is!